components/brave_wallet/renderer/js_ethereum_provider.cc

```diff
@@ -153,14 +153,14 @@ bool JSEthereumProvider::EnsureConnected() {
 void JSEthereumProvider::Install(bool allow_overwrite_window_ethereum_provider,
                                  content::RenderFrame* render_frame) {
   v8::Isolate* isolate = blink::MainThreadIsolate();
-  v8::MicrotasksScope microtasks(isolate,
-                                 v8::MicrotasksScope::kDoNotRunMicrotasks);
```

Comment:
The changing of the order here might cause a problem- I don't know if we need to move the `handle_scope` down also? wasn't sure

Reply:
Should be fine.